src/main/scala/uk/gov/nationalarchives/FileProcessor.scala

```diff
@@ -86,9 +86,13 @@ class FileProcessor(
           List(IdField("URI", parsedUri.get.uriWithoutDocType))
         )
 
-    val folderMetadataIdFields = potentialCite
-      .map(cite => List(IdField("Code", cite), IdField("Cite", cite)) ++ uriIdField)
+    val folderMetadataIdFields = department
```

Comment:
Do you think it's worth calling `department`, `potentialDepartment`? (I guess the same with `series`)?

Reply:
Yeah, that matches elsewhere where `Option` is called potentialXxxx I've changed it.